oca_dependencies.txt

```diff
@@ -1,2 +1,3 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
 delivery-carrier
+server-tools
```

Comment:
we use PIP mode https://github.com/OCA/sale-promotion/blob/13.0/.copier-answers.yml#L5
You can trash this file.

Reply:
ah, funny... on v13 PIP, on v14 OCA https://github.com/OCA/sale-promotion/blob/14.0/.copier-answers.yml

can we switch it?

Reply:
- https://github.com/OCA/sale-promotion/pull/61

Reply:
done